integration/reconfiguration_test.go

```diff
@@ -163,6 +163,51 @@ func TestReconfiguration_ExplicitStreams(t *testing.T) {
 	doReconfigurationTest(t, state, input, output)
 }
 
+func TestReconfiguration_StreamFileDoesNotExist(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %v", err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	nonExistentFile := filepath.Join(tempDir, "non-existent/fifo")
+
+	tests := []struct {
+		flag       string
+		wantStderr string
+	}{
+		{
+			"--input=" + nonExistentFile,
+			fmt.Sprintf("Unable to open file \"%s\" for reading: "+
+				"open %s: no such file or directory",
+				nonExistentFile, nonExistentFile),
+		},
+		{
+			"--output=" + nonExistentFile,
+			fmt.Sprintf("Unable to open file \"%s\" for writing: "+
+				"open %s: no such file or directory",
+				nonExistentFile, nonExistentFile),
+		},
+	}
+
+	mnt := filepath.Join(tempDir, "mountPoint")
+	mkdirAllOrFatal(t, mnt, 0755)
+	defer os.RemoveAll(mnt)
+
+	for _, test := range tests {
```

Comment:
Please use `t.Run()` to run subtests so that they are independent.

Also please be consistent with the naming and style used everywhere else. `cli_test.go` and `signal_test.go` contain a bunch of samples.

Reply:
Okay..
In the meantime could you check why Travis is still failing on OSX? I don't have a Mac to test the code.

Reply:
I can do the proposed changes if you want prior internal review+checkin, and test them against macOS manually. Let me know.

Reply:
I think I have already done the changes. But please do test them on macOS manually. I have no way to do so, and Travis build queues are too long to be of much use.